lib/modules/datasource/conda/index.ts

```diff
@@ -43,12 +48,29 @@ export class CondaDatasource extends Datasource {
       return null;
     }
 
+    if (registryUrl.startsWith('https://prefix.dev/')) {
+      const channel = ensureTrailingSlash(registryUrl).split('/').at(-2);
+      /* v8 ignore next 8: manager extract conda packages with unexpected registryUrl */
```

Comment:
please add a test instead

Reply:
for example, even it's wrong url with `https://prefix.dev/` , the `channel` will still be a string